src/App.tsx

```diff
@@ -1,26 +1,73 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import { useCallback, useEffect, useState } from 'react';
+import { getTodos } from './api';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { todosSlice } from './features/todos';
+import { Todo } from './types/Todo';
+import { Status } from './types/Status';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App = () => {
+  const todo = useAppSelector(state => state.currentTodo);
+  const dispatch = useAppDispatch();
+  const [isLoading, setIsLoading] = useState(false);
+  const filteres = useAppSelector(state => state.filter);
 
-          <div className="block">
-            <TodoFilter />
-          </div>
+  const filteredTodo = useCallback(
```

Comment:
```suggestion
  const filteredTodos = useMemo(
```
Just filter the todos that you have in the state, store in a variable and then show only them